es/wallets.md

```diff
@@ -32,15 +32,15 @@ Como puedes ver, hay una hay gran variedad para las distintas combinaciones usad
 
-Digamos que Alice tiene 3 notas de 10 dólares en su cartera, y ella quiere comprar 50 manzanas que Bob vende a un costo de 18 dólares.
-Alice le paga a Bob usando ambas notas de 10 dólares, y recibe 2 dólares de cambio (los cuales ella no tenía antes) como una nueva salida de transacción no gastada.
-Por lo tanto, Alice, ahora tiene una nota de 10 dólares y un cambio de 2 dólares  - añadidos a su cartera. Revisar su cartera por notas anteriores resultará en una nota de 10 dólares - pero el total de fondos en su cartera incluirán los 2 dólares de cambio.
+Digamos que Alice tiene 3 billetes de 10 dólares en su cartera, y ella quiere comprar 50 manzanas que Bob vende a un costo de 18 dólares.
+Alice le paga a Bob usando dos billetes de 10 dólares, y recibe 2 dólares de cambio (los cuales ella no tenía antes) como una nueva salida de transacción no gastada.
+Por lo tanto, Alice, ahora tiene un billete de 10 dólares y un cambio de 2 dólares - añadidos a su cartera. Si ahora Alice revisa su cartera buscando billetes anteriores encontrará un billete de 10 dólares - pero el total de fondos en su cartera incluirán también los 2 dólares de cambio.
 
-De manera similar, cuando efectúas una transacción desde tu billetera - las entradas seleccionadas normalmente excederán las salidas y un cambio deberá de ser creado.
-Cuando consultes tus fondos, si estás viendo el valor de una dirección - pueda que no veas todos sus detalles, porque la billetera contiene varias direcciones, y los usuarios puede que no sepan cuántos fondos contiene cada dirección luego de efectuar transacciones.
-Por lo tanto, la mejor manera de consultar tus fondos es utilizando tu billetera software, la cual tuvo acceso a tus llaves. Revisar si tus fondos fueron recibidos en la dirección X, el explorador es bastante conveniente.
+De manera similar, cuando efectúa una transacción desde su billetera - las entradas seleccionadas normalmente excederán las salidas y un cambio deberá de ser creado.
+Cuando consultes sus fondos, si está viendo el valor de una dirección - puede que no vea todos sus detalles, porque la billetera contiene varias direcciones y los usuarios puede que no sepan cuántos fondos contiene cada dirección luego de efectuar transacciones.
+Por lo tanto, la mejor manera de consultar sus fondos es utilizando su billetera software, la cual tuvo acceso a sus llaves. El uso de un explorador blockchain es bastante conveniente para revisar si sus fondos fueron recibidos en la dirección X.
 
-Para más detalles, puedes leer el gran [blog de Emurgo](https://emurgo.io/en/blog/blockchain-primer-cardanos-utxo-model-simply-explained) el cual entre en más detalles sobre el modelo UTXO.
+Para más detalles, puede leer el excelente [blog de Emurgo](https://forum.cardano.org/t/manual-blockchain-el-modelo-utxo-de-cardano-explicacion-sencilla/28269) el cual provee más información sobre el modelo UTXO.
```

Comment:
I have changed the link to the original article using the link to the article translated into Spanish.